roles/openshift_logging/tasks/main.yaml

```diff
@@ -12,6 +12,14 @@
 
 - debug: msg="Created temp dir {{mktemp.stdout}}"
 
+- name: Create local temp directory for doing work in
```

Comment:
This is a direct move from the jks tasks.  We make it local to the control node because thats the only way we have access to certain generated files, at least that was my experience.  <soapbox> I found this is one of the more confusing parts of ansible.  Seems some things need/do run locally or have local dependencies and others do not and its not obvious what is what. </soapbox>